docs/getting-started/create.md

```diff
@@ -1,148 +1,91 @@
+menu: 1. Getting started
+metaTitle: Getting started with GraphCommerce
 ---
 
-# Create a GraphCommerce storefront
+# Getting started with GraphCommerce
 
-You're ready to create a GraphCommerce storefront. You want to set up your
-development environment so that you can begin development.
-
-In this tutorial, you'll create a GraphCommerce app locally to begin developing
-a full-featured storefront. GraphCommerce is a front-end framework used for
-building headless Magento e-commerce PWA's.
-
-### What you'll learn
-
-After you've finished this tutorial, you'll have accomplished the following:
-
-- Set up your local development environment
-- Generated a new project based on the magento-graphcms example
-
-<figure>
-
-![GraphCommerce magento-graphcms example category page](https://user-images.githubusercontent.com/1251986/157831693-7ef3a2fb-779c-406e-8dd6-b984135ec58c.png)
-
- <figcaption>GraphCommerce magento-graphcms example category page</figcaption>
-</figure>
-
-<figure>
-
-![GraphCommerce magento-graphcms example product page](https://user-images.githubusercontent.com/1251986/157831746-461cd0cf-8671-4780-bafc-ae853f3f93da.png)
-
- <figcaption>GraphCommerce magento-graphcms example product page</figcaption>
-</figure>
+In this guide, you will set up a GraphCommerce app locally allowing you to start
+building.
 
 ### Requirements
 
-You've familiarized yourself with
-[React ↗](https://reactjs.org/docs/getting-started.html),
-[Next.js ↗](https://nextjs.org/docs/getting-started), and
-[Mui ↗](https://mui.com/material-ui/getting-started/overview/). GraphCommerce is
-a front-end React framework that uses Next.js for server-side rendering.
-
-### Install dependencies
-
-If you want to test a GraphCommerce storefront using a pre-configured Magento
-demo store and a pre-configured Hygraph project with demo content, then you need
-to only install the dependencies. This is the quickest approach.
-
 - Install and use node 16/18: `nvm install 16` or `nvm use 16`
 - Install yarn: `corepack enable`
 
-## Step 1: Create a new GraphCommerce app
+## Step 1: Create a GraphCommerce app
 
-<figure>
+```bash
+git clone -b main --depth 1 https://github.com/graphcommerce-org/graphcommerce.git
+# Clone repository
+```
 
-https://user-images.githubusercontent.com/1251986/158647122-dc57002f-a9c2-4661-a0a0-28a868ecf2f8.mp4
+```bash
+mkdir my-project
+# Create project folder
+```
 
-<video width="100%" controls autoPlay loop muted playsInline>
-<source src="https://user-images.githubusercontent.com/1251986/158647122-dc57002f-a9c2-4661-a0a0-28a868ecf2f8.mp4" type="video/mp4"/>
-</video>
+```bash
+cp -R graphcommerce/examples/magento-graphcms/. my-project && rm -rf graphcommerce && cd my-project
+# Copy example, delete repo, navigate to project folder
+```
 
-  <figcaption>GraphCommerce installation</figcaption>
-</figure>
+## Step 2: Configure API keys
 
-### Download the example
+Rename the configuration example file to: `graphcommerce.config.js` and
```

Comment:
If you do not have a graphcommerce.config.js file in your project it will start in demo mode and everything should be running as expected.

Reply:
Suggest copying the example instead of renaming? `cp graphcommerce.config.js.example graphcommerce.config.js`